### PR TITLE
Convert cross-signing feature flag to setting

### PR DESCRIPTION
**Pull request by turt2live**
This is intended as a temporary measure until we're comfortable with removing the flag entirely (something like https://github.com/matrix-org/matrix-react-sdk/pull/4417).

### CODE REVIEW

File: src/components/views/settings/tabs/user/LabsUserSettingsTab.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ export default class LabsUserSettingsTab extends React.Component {
                 </div>
                 <div className="mx_SettingsTab_section">
                     {flags}
+                    <SettingsFlag name={"feature_cross_signing"} level={SettingLevel.DEVICE} />
```
Comment (turt2live) [inline src/components/views/settings/tabs/user/LabsUserSettingsTab.js:65]: We'd remove this if we didn't want the toggle exposed.

File: src/settings/Settings.js [hunk 2]
```diff
@@ -152,10 +152,10 @@ export const SETTINGS = {
         default: null,
     },
     "feature_cross_signing": {
-        isFeature: true,
+        //isFeature: true,
         displayName: _td("Enable cross-signing to verify per-user instead of per-session (in development)"),
         supportedLevels: LEVELS_FEATURE,
-        default: false,
+        default: true,
```
Comment (jryans) [inline src/settings/Settings.js:158]: I know we've been down this road before, but I can't remember the answer, sorry... Why is it not enough to change just this one line here? In limited testing, it does seem to do the right thing, and it's still possible to override from a config file if desired.

File: src/components/views/settings/tabs/user/LabsUserSettingsTab.js (hunk 1 above)
Comment (turt2live) [reply src/components/views/settings/tabs/user/LabsUserSettingsTab.js:65]: ftr: we don't want it exposed, so will remove.

**Comment by turt2live** [comment]
@matrix-org/product please review this for the theory of bringing cross-signing out of labs and into the real world.

**Comment by turt2live** [comment]
@jryans this should be ready for review. I've tested it and it enables it by default for a new test account, which managed to cross-sign and verify the real me.

**Comment by turt2live** [comment]
(the nightmare of test fixing has been approved oob)

**Pull request merged**